main.go

```diff
@@ -22,6 +22,7 @@ import (
 )
 
 const framesHz = 9 // approx
+const cptvTempGlob = "*.cptv.temp"
```

Comment:
It might be good to have a const for the `.cptv.temp` extension and then use the to create the glob as well as using it in `newRecordingTempName()`. That there's no chance the deleting code and the creating code can be using different extensions.